ui/app/utils/classes/log.js

```diff
@@ -10,6 +11,8 @@ import PollLogger from 'nomad-ui/utils/classes/poll-logger';
 
 const MAX_OUTPUT_LENGTH = 50000;
 
+export const fetchFailure = url => () => Ember.Logger.warn(`LOG FETCH: Couldn't connect to ${url}`);
```

Comment:
Ember.Logger's is being deprecated and removed (not that it matters here or will happen soon), but console.warn is likely fine here too.

Reply:
Yeah... Not gonna lie, I just wanted to avoid adding an eslint-ignore.

Reply:
lol, fair